partiql-parser/src/peg.rs

```diff
@@ -33,10 +33,7 @@ impl<'val, R: RuleType> PairsExt<'val, R> for Pairs<'val, R> {
                 }
                 Ok(pair)
             }
-            None => syntax_error(
-                "Expected at one token pair, got nothing!",
-                Position::Unknown,
-            ),
+            None => illegal_state("Expected at one token pair, got nothing!"),
```

Comment:
Not your fault, but the message doesn't seem right. Should it be "Expected at _least_ one" or "Expected ~~at~~ one"?

Reply:
This one is my fault, I can clean it up.

```suggestion
            None => illegal_state("Expected at least one token pair, got nothing!"),
```